osc/commandline.py

```diff
@@ -5250,6 +5250,8 @@ def do_browse(self, subcmd, opts, *args):
                         help='Use server side generated sources instead of local generation.')
     @cmdln.option('-l', '--limit-size', metavar='limit_size',
                         help='Skip all files with a given size')
+    @cmdln.option('--native-obs-package', action='store_true',
+                        help='Do not clone native scm repositories: Different representation and you will not be able to submit changes!')
```

Comment:
overindented (should be one space beyond the opening parentheses)

Reply:
yes, but I don't want to become inconsistent with all other options ... let's either change all or none  (but I would do it in another commit)